plugins/woocommerce/client/blocks/tests/e2e/tests/utils/register-block-single-product-template.block_theme.spec.ts

```diff
@@ -129,6 +129,51 @@ test.describe( 'registerProductBlockType registers', () => {
 		} );
 	} );
 
+	test( 'blocks which are registered via the registerProductBlockType function are visible in the templates data views', async ( {
+		admin,
+		page,
+	} ) => {
+		const productBlockTypes = [
+			'woocommerce/product-price',
+			'woocommerce/product-rating',
+		];
+
+		await admin.visitAdminPage(
```

Comment:
I used `visitAdminPage` instead of `visitSiteEditor` to avoid going through the steps of loading the templates view. `path` option param wasn't returning expected results in the CI vs locally.